projects/frontend/shared-components/gui/projects/shared/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@vdk/shared",
+  "version": "3.0.0",
+  "engines": {
+    "node": "^14.15.0 || >=16.10.0",
+    "npm": ">=8.5.5"
+  },
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@angular/animations": "~13.3.10",
+    "@angular/cdk": "~13.3.8",
+    "@angular/common": "~13.3.10",
+    "@angular/compiler": "~13.3.10",
+    "@angular/core": "~13.3.10",
+    "@angular/forms": "~13.3.10",
+    "@angular/platform-browser": "~13.3.10",
+    "@angular/platform-browser-dynamic": "~13.3.10",
+    "@angular/router": "~13.3.10",
+    "@cds/core": "~5.7.2",
+    "@clr/angular": "~13.3.1",
+    "@clr/icons": "~13.0.2",
+    "@clr/ui": "~13.3.1",
+    "@ngrx/effects": "~13.2.0",
+    "@ngrx/router-store": "~13.2.0",
+    "@ngrx/store": "~13.2.0",
+    "@swimlane/ngx-graph": "~8.0.0",
+    "@webcomponents/custom-elements": "~1.5.0",
+    "angular-ng-autocomplete": "~2.0.8",
+    "d3": "~5.16.0",
+    "dompurify": "~2.3.8",
+    "lodash": "~4.17.21",
+    "ngx-clipboard": "~15.1.0",
+    "ngx-cookie-service": "~13.2.1",
+    "rxjs": "~7.5.5",
+    "zone.js": "~0.11.5"
+  },
+  "dependencies": {
+    "tslib": "2.3.1"
+  },
+  "description": "Taurus Shared library enables reusability of shared features like: NgRx Redux, Error Handlers, Utils, Generic Components etc.",
+  "repository": {
+    "type": "git",
+    "url": "-"
+  },
+  "keywords": [
+    "taurus",
+    "shared"
```

Comment:
a change of keywords?
I think of adding "versatile-data-kit", "vdk" - "taurus" could be cleared up (is was the name before open-sourcing the component)